.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,106 @@
+name: Publish to PyPI
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - main
```

Comment:
```suggestion
```

@Revathyvenugopal162 if this is a push to pypi only workflow, then *i think* we can remove the trigger on push to main

Reply:
sure, i adapted this from the https://github.com/pyOpenSci/pyosMeta/blob/main/.github/workflows/publish-pypi.yml, i can revert when pushing with a tag

Reply:
i am sorry - you ar e right

if: github.event_name == 'release' 

that ensures it only publishes to pypi on a release.